src/Framework/Framework/Resources/Scripts/postback/postbackCore.ts

```diff
@@ -202,16 +202,6 @@ function updateDynamicPathFragments(context: any, path: string[]): void {
     }
 }
 
-function processPassedId(id: any, context: any): string {
```

Comment:
:thinking: Wondering how postbacks in markup control in client-rendered repeater work without this

Reply:
This is being tracked in a separate issue #1218.